static/js/compose_ui.js

```diff
@@ -141,6 +141,13 @@ export function compute_placeholder_text(opts) {
 export function wrap_text_with_markdown(textarea, prefix, suffix) {
     const range = textarea.range();
 
+    if (prefix === "**" || prefix === "*") {
+        const rangeLength = range.length;
+        if (range.text.charCodeAt(rangeLength - 1) === 10) {
+            range.text = range.text.slice(0, -1);
```

Comment:
This confusingly mutates the object returned by `$.fn.range()`, which isn’t our object to mutate. Use a local variable instead.

Reply:
Is the mutation is caused due to ```range.text.slice(0, -1);``` or the declared ```range``` constant ?

Reply:
When you write `range.text =`, you are mutating `range`.

Reply:
I have made the changes @andersk , Can you please review it